### PR TITLE
Configures TLS enabled `docker` daemon

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes https://github.com/projectatomic/adb-atomic-developer-bundle/issues/85

### CODE REVIEW

**Comment by bexelbie** [comment]
LGTM

File: docs/using.rst [hunk 1]
```diff
@@ -60,6 +63,29 @@ The `helloapache <https://registry.hub.docker.com/u/projectatomic/helloapache/>`
...
+
+Today, the only support is for the exposure of the docker daemon port.  The docker daemon is TLS protected, so in addition to exposing the port you must configure the docker command on the host to use TLS and the right port.  This can be done via environment variables.
+
+1. Install the ``vagrant-adbinfo`` plugin in order to get easy access to the correct environment variables and the TLS certificates.
+
+   ``vagrant plugin install vagrant-adbinfo``
+
+2. If you haven't already, modify your Vagrantfile to expose the docker daemon port by adding this line:
+
+   config.vm.network "forwarded_port", guest: 2376, host: 2379, auto_correct: true
```
Comment (navidshaikh) [inline docs/using.rst]: how about adding the the line in code block?

```
config.vm.network "forwarded_port", guest: 2376, host: 2379, auto_correct: true
```

**Comment by navidshaikh** [comment]
LGTM! :+1:

File: docs/architecture.rst [hunk 2]
```diff
@@ -38,7 +38,14 @@ These boxes contains the following components from the latest CentOS Core:
...
   * atomic - The `Atomic CLI <https://github.com/projectatomic/atomic>`_
   * docker-registry
-  * docker - The docker daemon is configured to expose a TLS protected TCP port so that vagrant host based tools can access it
+  * docker
+    
+    The docker daemon is configured to expose a TLS protected TCP port so that vagrant host based tools can access it.  This is accomplished in three steps:
+
+    1. The systemd startup for docker inside of the ADB will generate TLS certificates if they don't already exist.
+    2. The user of the ADB exposes the docker daemon port in their Vagrantfile
+    3. The certificates are copied out by the vagrant-adbinfo plugin.
```
Comment (LalatenduMohanty) [inline docs/architecture.rst]: I think we should put the link to vagrant-adbinfo project.

**Comment by navidshaikh** [comment]
Looks good to me after squashing! :+1:

**Comment by bexelbie** [comment]
Updated with a second link to vagrant-adbinfo

**Comment by LalatenduMohanty** [comment]
@bexelbie @navidshaikh LGTM, thanks :)

**Pull request merged**